CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 * **[Feature]** Increase the interval between sending logs from 3 to 6 seconds for the backend load optimization.
 * **[Feature]** Add `Analytics.enableManualSessionTracker` and `Analytics.startSession` APIs for tracking session manually.
+* **[Feature]** Add `AppCenter.setLogger` API to set custom logger for logs output.
```

Comment:
Let's remove ` for logs output` part. Just "Add `AppCenter.setLogger` API to set a custom logger."